Tasks/Ganus/task3_3.py

```diff
@@ -0,0 +1,6 @@
+str_enter = input("Введите, пожалуйста, строку\n")
+str_enter = str_enter.lower()
+str_enter = str_enter.replace(',', '').replace('.', '').replace(':', '').replace('-', '').replace('!', '').replace('?', '')
+str_enter = str_enter.split(" ")
+letter_count = {letter: str_enter.count(letter) for letter in str_enter}
```

Comment:
this is a clever way to do it, I must say, BUT it's not optimal, you're going through each of letters and you will count some of them several times, like if you have string "ABBA" you will count the "A" appearances for the first letter, and then you will do the same operation for the last one